bin/run-tests

```diff
@@ -26,6 +26,8 @@ for D in exercises/*; do
           make clean >> /dev/null;
           if make | grep FAIL: ; then
             exit 1
+          elif [ ${PIPESTATUS[0]} -ne 0 ]; then
```

Comment:
I assume this works, but couldn't you simply add `set -e` to the top of this script so it fails if anything goes wrong?